web/viewer.css

```diff
@@ -15,11 +15,79 @@
 
 @import url(pdf_viewer.css);
 
+/** ############### VARIABLES ############### **/
+
+/* ------------ Initial ------------ */
+
 :root {
   --sidebar-width: 200px;
-  --sidebar-transition-duration: 200ms;
-  --sidebar-transition-timing-function: ease;
-}
+
+  --main-color: #0c0c0d;
```

Comment:
Can you please use a *consistent* colour format throughout the patch, rather than mixing formats? 

There's at least four different formats used below, e.g. `#0c0c0d;`, `#5C5C61;`, `rgb(255, 74, 74);`, and `rgba(245, 246, 247, 1);`; please note PR #11447.